packages/react-devtools-extensions/chrome/manifest.json

```diff
@@ -40,7 +40,7 @@
     "persistent": false
   },
 
-  "permissions": ["file:///*", "http://*/*", "https://*/*"],
+  "permissions": ["file:///*", "http://*/*", "https://*/*", "contextMenus"],
```

Comment:
New permission added only to the Chrome manifest? How does this impact Firefox and Edge?

Can we add this new permission as an [`optional_permissions`](http://www.adambarth.com/experimental/crx/docs/permissions.html) instead?

Maybe we could add an item to the settings menu, e.g. "Enable context menu", that prompts the user to grant the permission. It's not as convenient or discoverable but I'm wary about adding new required permissions. Might cause problems or complaints for people updating.

Reply:
In this PR, there is no impact on Firefox and Edge. Target was only chrome. Should I add it for them too?
Let me update optional_permissions.

Reply:
Yes that's true. I think chrome will disable the extension, saying that the extension has been disabled as it requires additional permissions. Have to check if contextMenus can be requested as an optional permission. I think it should be

Reply:
@sktguha any update?

Reply:
Oh hi @rickhanlonii , actually I didn't work on this, I was just giving some PR comments.
I think @asheshvidyut7 is the author of the PR.

But I think contextMenu permission can be an optional permission as per the text below
https://developer.chrome.com/docs/extensions/reference/permissions/#step-2-declare-optional-permissions-in-the-manifest:~:text=Most%20Chrome%20extension%20permissions%20can%20be%20specified%20as%20optional%2C%20with%20the%20following%20exceptions.